syntax = "proto2";
import "serverinfo_card.proto";
import "zone_type.proto";

message ServerInfo_Zone {
    optional string name = 1;
    optional ZoneType type = 2;
    optional bool with_coords = 3;
    optional sint32 card_count = 4;
    repeated ServerInfo_Card card_list = 5;
    // Reveal top card to all players.
    optional bool always_reveal_top_card = 10;
    // reveal top card to the owner.
    optional bool always_look_at_top_card = 11;
}
